app/routes/web.py

```
@@ -403,3 +403,12 @@ def generate_api_key():
         flash("An error occurred while generating your API key. Please try again.", "danger")
     
     return redirect(url_for('web.profile'))
+
+# Test Sentry
+@web.route("/error")
+def hello():
+    division_by_zero = 1 / 0
```

Ruff (0.8.2)

Local variable `division_by_zero` is assigned to but never used

Remove assignment to unused variable `division_by_zero`

(F841)